js/coinsbit.js

```diff
@@ -0,0 +1,1103 @@
+'use strict';
+
+// ---------------------------------------------------------------------------
+
+const Exchange = require ('./base/Exchange');
+const { ExchangeError, ArgumentsRequired, InvalidOrder, OrderNotFound } = require ('./base/errors');
+const Precise = require ('./base/Precise');
+
+// ---------------------------------------------------------------------------
+
+module.exports = class coinsbit extends Exchange {
+    describe () {
+        return this.deepExtend (super.describe (), {
+            'id': 'coinsbit',
+            'name': 'Coinsbit',
+            'countries': [ 'EE' ], // Estonia <<<TODO>>> they also have .in domain
```

Comment:
Let's write `[ 'IN', 'EE' ]`.

Reply:
so, should we  mention 'India' in comment part or not?  also, are we sure https://coinsbit.in/is same coinsbit exchange/company ? I've not found link betwen each other on main site/social profies. Just found 'india' version from google.

Reply:
Yes, after a bit of googling for _"Coinsbit Estonia India"_ I found a few articles referencing them as a parent-child company with headquarters in Estonia. 

We should add the `hostname` functionality to it, so that it would default to the `.io` address and the users would be able to use the `.in` address as well. Find all occurrences of `'hostname'` around the exchanges in `describe()` and in `sign()`.

I think they're two different websites, with two different userbases, different sets of markets, hence they're two different exchanges in one. We should support both in this integration.